skorch/callbacks/training.py

```diff
@@ -87,6 +94,9 @@ def on_epoch_end(self, net, **kwargs):
                     last_epoch=net.history[-1],
                     last_batch=net.history[-1, 'batches', -1],
                 )
-            if net.verbose > 0:
-                print("Checkpoint! Saving model to {}.".format(target))
+            self._sink("Checkpoint! Saving model to {}.".format(target), net.verbose)
             net.save_params(target)
+
+    def _sink(self, text, verbose):
+        if (self.sink is not print) or verbose:
```

Comment:
Why check if the sink is not print? I would only expect a verbosity check here. Also I woild move the reference of net.verbose to this method since it is unlikely to be something different.

Reply:
The check was directly copy/pasted from `PrintLog`. I don't quite understand why, but I assumed there was some clever reason.

Regarding moving the reference, `net` is not part of self, so it's not accessible in `_sink`. Or do I misunderstand your proposal?

Reply:
The idea here is that if sink is not print, you don't want it to be affected by verbosity.

Reply:
Ah, my bad. This is probably fine then. But since 2 out of 3 missed the purpose, adding an inline comment might be appropriate.

You are right about `net.verbose` of course.